docs/06_how-to-guides/20_authorization/how_to_restrict_access_to_an_action_by_user.md

```diff
@@ -79,4 +79,4 @@ void hi( name user ) {
 
 ## Summary
 
-In conclusion, the above instructions show how to how to perform authorization checks in a smart contract.
+In conclusion, the above instructions show how to perform authorization checks in a smart contract.
```

Comment:
perhaps we should discuss the summary in the team meeting so that we standardise on a 'format'.

I have used:

## Summary
Follow these instructions to purchase RAM, with a specified amount of tokens, for the specified accounts.

Basically defining what the how to showed ... happy to change this to a more suitable summary but I think we should 'standardise' the language so they are consistent for all the how to's

Reply:
yes I agree . I used the format that @sunshineveg and @bobgt gave us a couple of weeks ago.